### PR TITLE
feat added playlist

### PR DESCRIPTION
**Pull request by AmishMishra11**
Added feature to create a new playlist
Added feature to delete the playlist
Added feature to add videos to the playlist
Added feature to remove a video from the playlist
Responsiveness

https://user-images.githubusercontent.com/76205249/166163088-4730244b-0b21-495f-8bf4-c8151f606d01.mp4

### CODE REVIEW

File: src/Components/ModalPlaylist/ModalPlaylist.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  const addPlaylistVideoHandler = (id) => {
+    addPlaylistVideo(id, item, dispatchVideo);
+  };
+
+  const removePlaylistVideoHandler = (id) => {
+    removePlaylistVideo(id, item._id, dispatchVideo);
+  };
+
+  // const [alreadyInPlaylist, setAlreadyInPlaylist] = useState(false);
```
Comment (VedanthB) [inline src/Components/ModalPlaylist/ModalPlaylist.jsx:27]: remove commented code if you're not going to use it.

**Review by VedanthB — APPROVED** [review]
Good work

things i liked 

- the code is clean ,readable and well organised 
- css spacing units used are consistent 

Suggestion:
- maybe use a lighter shade of red this seems too bright.

you can merge the PR

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean
I liked that you have made separate files for each functionality

File: src/Components/ModalPlaylist/ModalPlaylist.jsx (hunk 1 above)
Comment (AmishMishra11) [reply src/Components/ModalPlaylist/ModalPlaylist.jsx:27]: Thanks

**Pull request merged**